### DISCUSSION TITLE
目前V2版预览版好像还是用着v2这分支的插件?

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#2565] [提问 / Q&A]
好几个preview分支修复了的bug
2.0.2预览版手动更新了还是存在

**Comment by the1812** [comment]
从 TP 版导出的设置里功能可能还是指向 `v2` 分支的, 要接收最新更新, 可以安装 `更新链接替换` 组件, 然后在功能面板里启动, 输入分支名 `preview` 就可以把已安装功能的更新链接分支切换过来. 弄完之后可以删掉 `更新链接替换`.
